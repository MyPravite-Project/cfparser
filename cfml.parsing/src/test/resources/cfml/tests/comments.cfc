//this is a test comment
var x = 123;
var y = x;