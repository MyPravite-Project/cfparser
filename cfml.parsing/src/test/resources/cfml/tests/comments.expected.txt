/*===TOKENS===*/
LINE_COMMENT             <//this is a test comment
>
VAR                      <var>
IDENTIFIER               <x>
'='                      <=>
INTEGER_LITERAL          <123>
';'                      <;>
VAR                      <var>
IDENTIFIER               <y>
'='                      <=>
IDENTIFIER               <x>
';'                      <;>
/*===TREE===*/
(scriptBlock
  (element
    (statement
      (localAssignmentExpression
        var
        (identifier x)
        =
        (startExpression
          (compareExpression
            (baseExpression (unaryExpression (primaryExpression (literalExpression 123))))
          )
        )
      )
      ;
    )
  )
  (element
    (statement
      (localAssignmentExpression
        var
        (identifier y)
        =
        (startExpression
          (compareExpression
            (baseExpression (unaryExpression (memberExpression (identifier x))))
          )
        )
      )
      ;
    )
  )
  (endOfScriptBlock <EOF>)
)
/*======*/